packages/app/src/app/pages/common/Navigation/Notifications/notifications/TeamAccepted.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { inject } from 'mobx-react';
+
+import { NotificationImage as Image } from '../elements';
+import { Container, W } from './elements';
+
+interface Props {
+  read: boolean;
+  teamName: string;
+  userName: string;
+  userAvatar: string;
+}
+
+const TeamInvite = ({ read, teamName, userName, userAvatar }: Props) => {
```

Comment:
@SaraVieira This can be an implicit return

```js
const TeamInvite = ({ read, teamName, userName, userAvatar }: Props) => (
  // ...
);
```

Reply:
True, changed :)